src/App.tsx

```diff
@@ -37,6 +42,17 @@ function NavLink(props: NavProps): JSX.Element {
   );
 }
 
+function Copyright(_props: RouteComponentProps): JSX.Element {
```

Comment:
Some Typescript help...

RouteComponentProps comes from Reach Router and adds the 'path' parameter. It is used on page-level components that you expect to navigate to. Since Copyright isn't a navigable target, it shouldn't take RouteComponentProps. It can just take a straight object in most cases. I like to declare it as a type right above the function, like this:
```
type Props = { // You could use a different name here to avoid name collisions
  prop1: string;
  prop2: number;
};
function MyComponent(props: Props): JSX.Element { ... }
```

Good job on figuring out you need the leading '_' on unused parameters! On the other hand, since you don't use it and the function works without it, it's probably best to just leave off the parameter list entirely: `Copyright(): JSX.Element`. Unused parameters are pretty uncommon. They usually show up when you are forced to make a function parameter list look like one in a library.

Reply:
I will take off the parameter, thank you for the help! I will copy this and add to my note :-)